docs/contributing.md

```diff
@@ -1,8 +1,8 @@
 We'd love you to contribute to Pydantic!
+
 ## Issues
 
-Questions, feature requests and bug reports are all welcome as [discussions or issues](https://github.com/pydantic/pydantic/issues/new/choose). **However, to report a security
-vulnerability, please see our [security policy](https://github.com/pydantic/pydantic/security/policy).**
+Questions, feature requests and bug reports are all welcome as [discussions or issues](https://github.com/pydantic/pydantic/issues/new/choose). **However, to report a security vulnerability, please see our [security policy](https://github.com/pydantic/pydantic/security/policy).**
```

Comment:
Everything on the same line is not an improvement... 👀

Reply:
I agree with Marcelo, I prefer it be max width of 120 chars per line as much as possible